### ISSUE TITLE
Suspected BUG: Apps launch causing system reboot

### ISSUE THREAD

**Issue by Oolong666** [boxeehacks/boxeehack#326]
I just upgraded to the new version (v1.4.1) and when I try to lauch the apps module it displays an Error Script Fail! (boxeehack_system.py) and then reboots.  It is trying to lauch my "favorites" and although I tried to change this to "all" it doesn't allow interaction before rebooting.

I've set my homepage to lauch Netflix as my movies and that lauches properly so it doesn't appear to be the actual operation of the apps themselves just the apps menu/screen.

Apps are a major feature of my Boxee Box use so I really hope something can be done soon.

**Comment by Oolong666** [comment]
FYI: Have a couple of custom repositories

.... Sorry new to this, clicked the wrong button and closed by accident.

**Comment by boxeehacks** [comment]
Are you using a different version of Boxee+Hacks than the one from this github repo? Because in the software there exists no file called boxeehack_system.py ...

**Comment by Oolong666** [comment]
Not that I'm aware of.  I downloaded it from github.com/boxeehacks/boxeehack/archive/master.zip

I'm going to remove my repositories and try again.... perhaps it's coming from there.

**Comment by Oolong666** [comment]
Removed my custom repositories and set my default Apps view to "all".  Then I flushed all my caches etc. and am now able to get into the apps.  HOWEVER when I go to my favorites it still reboots and still claims "Error Script Fail!: boxeehack_system.py".

I didn't add anything else and have only used the hack from the Boxee+ community so it must be coming from somewhere in the scripts.

Can someone tell me how to manually remove all my favorites (I've already gone through each of the apps and selected "remove" I'm looking for a way to ensure there are no entries via the command line (telnet).  Maybe there is a legacy custom app that is causing an issue ... one I can see as I can't get into this screen.  Other than this I've run out of ideas.
